### PR TITLE
Do not localize deletion emails.

### LINKED ISSUE #909: Do not localize add-on deletion emails
https://github.com/mozilla/olympia/blob/master/apps/addons/models.py#L458 sends emails when an add-on is being deleted. The property `atype` is being localized. We don't want that as it makes it very hard to search in those emails.

I'd give a shot a this myself, but I don't know why l10n happens in the first place. Any hints are appreciated.

Example:
`The following ΘΈΜΑ was deleted.`

### PR DESCRIPTION
**Pull request by wagnerand**
This fixes #909

r?

### CODE REVIEW

File: apps/addons/models.py [hunk 1]
```diff
@@ -496,7 +496,9 @@ def delete(self, msg='', reason=''):
                         _current_version=None, guid=None)
             models.signals.post_delete.send(sender=Addon, instance=self)
 
-            send_mail(subject, email_msg, recipient_list=to)
+            # Don't localize email to admins, use 'en-US' always.
+            with user.activate_lang('en-US'):
+                send_mail(subject, email_msg, recipient_list=to)
```
Comment (magopian) [inline apps/addons/models.py]: I don't think you want to do that, but rather something like https://github.com/mozilla/olympia/blob/master/apps/editors/tasks.py#L60 (use https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.translation.override)

File: apps/users/models.py [hunk 2]
```diff
@@ -464,12 +464,12 @@ def special_collection(self, type_, defaults):
         return c
 
     @contextmanager
-    def activate_lang(self):
+    def activate_lang(self, override_lang):
```
Comment (magopian) [inline apps/users/models.py]: You need to revert the changes in this file, which aren't useful anymore I believe

**Comment by magopian** [comment]
Fixed the PR in #936. Thanks @wagnerand for investigating this!

File: apps/users/models.py (hunk 2 above)
Comment (EnTeQuAk) [inline apps/users/models.py]: Yeah, this makes travis fail too and you're not using it in this patch.
